### PR TITLE
feat(core): put application, taskWriter, executionService on window

### PR DESCRIPTION
**Pull request by anotherchrisberry**
Not sure if this is a terrible idea, but putting a few select items into the global context to allow simple console scripting for things like cancelling tasks and running pipelines.

With these in place, if you need to cancel a bunch of executions that have been running for more than four hours, you can just do something like this in the console:
```javascript
application.executions.data
  .filter(e => e.isRunning && e.runningTimeInMs > 4 * 60 * 60 * 1000)
  .forEach(e => executionService.cancelExecution(application, e.id, false, 'cancelled via console'));
```

Need to convert the taskWriter to TS and add it to the window to make this more robust but wanted to get feedback first.

### CODE REVIEW

**Comment by jrsquared** [comment]
seems reasonable. I would just remove `taskWriter` from the commit message.

File: app/scripts/modules/core/src/application/application.component.ts [hunk 1]
```diff
@@ -35,13 +35,15 @@ export class ApplicationController implements IComponentController {
     if (this.app.name.length > 20) {
       this.compactHeader = true;
     }
+    (window as any).application = this.app;
```
Comment (christopherthielen) [inline app/scripts/modules/core/src/application/application.component.ts]: Would it be better to namespace these? 

`spinnaker_application`
or
`spinnaker.application`

**Comment by anotherchrisberry** [comment]
Revised - now namespaced, made $injector available for anything not already declared to allow easier console scripting...

**Comment by anotherchrisberry** [comment]
another nice thing this lets you do:
```javascript
spinnaker.application.disableAutoRefresh();
```
to pause refreshes while debugging

**Pull request merged**